flict/__main__.py

```diff
@@ -266,6 +266,11 @@ def suggest_outbound_candidate(args):
 
 
 def policy_report(args):
+    try:
+        open(args.report_file).close()
+        open(args.policy_file).close()
+    except (FileNotFoundError):
+        flict_exit(ReturnCodes.RET_FILE_NOT_FOUND, "Provided file was not found.")
```

Comment:
It would be indeed nice to actually let the user know which file is not accessible

Reply:
done